### PR TITLE
Integrations to Packs - Phase 1

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
Phase 1 of moving 10 integrations from Legacy pack to own packs

### CODE REVIEW

**Comment by amshamah419** [comment]
@yuvalbenshalom Do we want to push this and then afterwards format the release notes to the new format?

**Comment by yuvalbenshalom** [comment]
@amshamah419 yes. we will need to do this change for many packs anyway.

File: Packs/AbuseDB/pack_meta.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AbuseIPDB",
+    "description": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (IkaDemisto) [inline Packs/AbuseDB/pack_meta.json:5]: ServerMInVersion should be server version and not pack version

File: Packs/AbuseDB/pack_meta.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AbuseIPDB",
+    "description": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration.",
+    "support": "xsoar",
```
Comment (amshamah419) [inline Packs/AbuseDB/pack_meta.json:4]:
```suggestion
    "support": "Cortex XSOAR",
```

File: Packs/AbuseDB/pack_meta.json (hunk 1 above)
Comment (Itay4) [reply Packs/AbuseDB/pack_meta.json:5]: it is the server version.. there is no `fromversion`

File: Packs/AbuseDB/pack_meta.json (hunk 2 above)
Comment (IkaDemisto) [reply Packs/AbuseDB/pack_meta.json:4]: should be `xsoar`, it is different from `author` field

Comment (amshamah419) [reply Packs/AbuseDB/pack_meta.json:4]: We have a few which are using this for support. We should change those then.

Comment (IkaDemisto) [reply Packs/AbuseDB/pack_meta.json:4]: Yes, and also change it in demisto-sdk if it was not changed yet

Comment (amshamah419) [reply Packs/AbuseDB/pack_meta.json:4]: I'll include it in my PR

Comment (IkaDemisto) [reply Packs/AbuseDB/pack_meta.json:4]: o.k I think it is defined here: https://github.com/demisto/etc/issues/23216#issuecomment-608375523

Comment (amshamah419) [reply Packs/AbuseDB/pack_meta.json:4]: sdk is quite out of date:

```
        metadata = {
            'name': '## FILL OUT MANUALLY ##',
            'description': '## FILL OUT MANUALLY ##',
            'support': 'demisto',
            'serverMinVersion': '## FILL OUT MANUALLY #',
            'currentVersion': PACK_INITIAL_VERSION,
            'author': 'demisto',
            'url': 'https://www.demisto.com',
            'email': '',
            'categories': [],
            'tags': [],
            'created': datetime.utcnow().strftime(Initiator.DATE_FORMAT),
            'updated': datetime.utcnow().strftime(Initiator.DATE_FORMAT),
            'beta': False,
            'deprecated': False,
            'certification': 'certified',
            'useCases': [],
            'keywords': [],
            'price': '0',
            'dependencies': {},
        }
```

I'll update the other fields while I'm at it.

**Comment by IkaDemisto** [comment]
@Itay4 there are several packs with typo of metadata file,  `pack_meta.json` instead of `pack_metadata.json`.

Full list:
```
@@@Issues with unique files in pack: AbuseDB
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: Tanium
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: CarbonBlackProtect
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: Cherwell
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: RedCanary
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: TaniumThreatResponse
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: IronDefense
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: FreshDesk
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: Devo_v2
  "pack_metadata.json" file does not exist, create one in the root of the pack
@@@Issues with unique files in pack: PolySwarm
  "pack_metadata.json" file does not exist, create one in the root of the pack
The files were found as invalid, the exact error message can be located above
```

**Comment by Itay4** [comment]
@yuvalbenshalom please have a look, i think we can force merge
im ok with the current failures in the validation step

File: Packs/AbuseDB/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "AbuseIPDB",
+    "description": "Central repository to report and identify IP addresses that have been associated with malicious activity online. Check the Detailed Information section for more information on how to configure the integration.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AbuseDB/pack_metadata.json:5]: please remove

File: Packs/AbuseDB/pack_metadata.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "AbuseIPDB"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AbuseDB/pack_metadata.json:20]: please remove

File: Packs/CarbonBlackProtect/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Carbon Black Enterprise Protection V2",
+    "description": "Carbon Black Enterprise Protection is a next-generation endpoint threat prevention solution to deliver a portfolio of protection policies, real-time visibility across environments, and comprehensive compliance rule sets in a single platform.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/CarbonBlackProtect/pack_metadata.json:5]: please remove

File: Packs/CarbonBlackProtect/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        "Endpoint"
+    ],
+    "tags": [
+        "Carbon Black Enterprise Protection V2"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/CarbonBlackProtect/pack_metadata.json:20]: please remove

File: Packs/Cherwell/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Cherwell",
+    "description": "Cloud-based IT service management solution",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Cherwell/pack_metadata.json:5]: please remove

File: Packs/Cherwell/pack_metadata.json [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+        "Case Management"
+    ],
+    "tags": [
+        "Cherwell"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Cherwell/pack_metadata.json:20]: please remove

File: Packs/Devo_v2/pack_metadata.json [hunk 9]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Devo v2",
+    "description": "Use the Devo v2 integration to query Devo for alerts, lookup tables, and to write to lookup tables.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Devo_v2/pack_metadata.json:5]: please remove

File: Packs/Devo_v2/pack_metadata.json [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+        "Analytics & SIEM"
+    ],
+    "tags": [
+        "Devo v2"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Devo_v2/pack_metadata.json:20]: please remove

File: Packs/FreshDesk/pack_metadata.json [hunk 11]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Freshdesk",
+    "description": "The Freshdesk integration allows you to create, update, and delete tickets; reply to and create notes for tickets as well as view Groups, Agents and Contacts.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/FreshDesk/pack_metadata.json:5]: please remove

File: Packs/FreshDesk/pack_metadata.json [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        "Case Management"
+    ],
+    "tags": [
+        "Freshdesk"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/FreshDesk/pack_metadata.json:20]: please remove

File: Packs/IronDefense/pack_metadata.json [hunk 13]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "IronDefense",
+    "description": "The IronDefense Integration allows users to interact with IronDefense alerts within Demisto. The Integration provides the ability to rate alerts, update alert statuses, add comments to alerts, and to report observed bad activity.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/IronDefense/pack_metadata.json:5]: please remove

File: Packs/IronDefense/pack_metadata.json [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+        "Network Security"
+    ],
+    "tags": [
+        "IronDefense"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/IronDefense/pack_metadata.json:20]: please remove

File: Packs/PolySwarm/pack_metadata.json [hunk 15]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "PolySwarm",
+    "description": "Real-time threat intelligence from a crowdsourced network of security experts and antivirus companies.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/PolySwarm/pack_metadata.json:5]: please remove

File: Packs/PolySwarm/pack_metadata.json [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "PolySwarm"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/PolySwarm/pack_metadata.json:20]: please remove

File: Packs/RedCanary/pack_metadata.json [hunk 17]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Red Canary",
+    "description": "Red Canary collects endpoint data using Carbon Black Response and CrowdStrike Falcon.  The collected data is standardized into a common schema which allows teams to detect, analyze and respond to security incidents.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/RedCanary/pack_metadata.json:5]: please remove

File: Packs/RedCanary/pack_metadata.json [hunk 18]
```diff
@@ -0,0 +1,26 @@
...
+        "Endpoint"
+    ],
+    "tags": [
+        "Red Canary"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/RedCanary/pack_metadata.json:20]: please remove

File: Packs/Tanium/pack_metadata.json [hunk 19]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Tanium",
+    "description": "Tanium endpoint security and systems management",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Tanium/pack_metadata.json:5]: please remove

File: Packs/Tanium/pack_metadata.json [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+        "Endpoint"
+    ],
+    "tags": [
+        "Tanium"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Tanium/pack_metadata.json:20]: please remove

File: Packs/TaniumThreatResponse/pack_metadata.json [hunk 21]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Tanium Threat Response",
+    "description": "Tanium Threat Response",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
+    "currentVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/TaniumThreatResponse/pack_metadata.json:6]: please remove

File: Packs/TaniumThreatResponse/pack_metadata.json [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+        "Endpoint"
+    ],
+    "tags": [
+        "Tanium Threat Response"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/TaniumThreatResponse/pack_metadata.json:20]: please remove

**Comment by Itay4** [comment]
@yuvalbenshalom done

**Comment by Itay4** [comment]
fails on non relevant validations
merging

**Pull request merged**